src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -222,11 +224,20 @@ trait FastSync {
         cleanupRequestedMaps(ref)
 
       case PrintStatus =>
-        val totalNodesCount = downloadedNodesCount + mptNodesQueue.size + nonMptNodesQueue.size
-        log.info(
-          s"""|Block: ${appStateStorage.getBestBlockNumber()}/${initialSyncState.targetBlock.number}.
-              |Peers: ${assignedHandlers.size}/${handshakedPeers.size} (${blacklistedPeers.size} blacklisted).
-              |State: $downloadedNodesCount/$totalNodesCount known nodes.""".stripMargin.replace("\n", " "))
+        printStatus()
+    }
+
+    private def printStatus() = {
+      val totalNodesCount = downloadedNodesCount + mptNodesQueue.size + nonMptNodesQueue.size
+      val formatPeer: (Peer) => String = peer => s"${peer.remoteAddress.getAddress.getHostAddress}:${peer.remoteAddress.getPort}"
+      log.info(
+        s"""|Block: ${appStateStorage.getBestBlockNumber()}/${initialSyncState.targetBlock.number}.
+            |Peers waiting_for_response/connected: ${assignedHandlers.size}/${handshakedPeers.size} (${blacklistedPeers.size} blacklisted).
+            |State: $downloadedNodesCount/$totalNodesCount nodes.
+            |Details: connected(${assignedHandlers.values.map(formatPeer).toSeq.sorted})/
```

Comment:
Isn't this the most important status printout? The one that lets the user know what's going on? If this was `debug` where would the user the notified of the status?